core/embed/projects/prodtest/cmd/secure_channel.h

```diff
@@ -0,0 +1,34 @@
+/*
+ * This file is part of the Trezor project, https://trezor.io/
+ *
+ * Copyright (c) SatoshiLabs
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "noise.h"
+
+#define SECURE_CHANNEL_OUTPUT_SIZE (sizeof(noise_response_t))
+#define SECURE_CHANNEL_INPUT_SIZE (sizeof(noise_request_t))
```

Comment:
I must be still confused about what is input/output. This looks like it should be the other way around.

Reply:
True, sorry. Fixed in https://github.com/trezor/trezor-firmware/pull/5283/commits/79634acd653b00badb2fda553dbec8f70dcc27da.